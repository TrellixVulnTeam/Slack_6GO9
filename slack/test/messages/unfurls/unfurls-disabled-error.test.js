const UnfurlsDisabledError = require('../../../lib/messages/unfurls/unfurls-disabled-error');

describe('UnfurlsDisabledError message rendering', () => {
  test('works', async () => {
    const message = new UnfurlsDisabledError();
    expect(message.getAttachment()).toMatchSnapshot();
  });
});
