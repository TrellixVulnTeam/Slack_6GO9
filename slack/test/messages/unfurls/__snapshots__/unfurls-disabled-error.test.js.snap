// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UnfurlsDisabledError message rendering works 1`] = `
Object {
  "color": "danger",
  "mrkdwn_in": Array [
    "text",
  ],
  "text": "Could not complete link preview because your workspace has disabled link previews for \`github.com\` <https://get.slack.help/hc/en-us/articles/204399343-Share-links-in-Slack#turn-off-link-previews-for-specific-sites|Read more>",
}
`;
